cmd/larry/main.go

```diff
@@ -95,6 +96,7 @@ func getProvider(cfg config.Config) (larry.Provider, error) {
 
 	cacheClient := cache.NewClient(ro)
 	if cfg.Provider == provider.Github {
+		blacklist.Load(cacheClient, cfg.BlacklistFile, cfg.GetCacheKeyPrefix())
```

Comment:
Can we grab the error from the Load method and log if err != nil?
Just to make easier to debug in case of Load failing

Reply:
Good catch - done!